lib/finch.ex

```diff
@@ -391,7 +391,7 @@ defmodule Finch do
 
   defp __stream__(%Request{} = req, name, acc, fun, opts) do
     {pool, pool_mod} = get_pool(req, name)
-    pool_mod.request(pool, req, acc, fun, opts)
+    pool_mod.request(pool, req, acc, fun, name, opts)
```

Comment:
What about having it `opts = Keyword.merge([pool_name: name], opts)`

Reply:
I like this one. It doesn't change arity of the funs and should make a step-by-step transition to use finch instance name in all events even smoother. @sneako thoughts?